sycl/doc/extensions/DataFlowPipes/data_flow_pipes_rev4_proposed.asciidoc

```diff
@@ -170,20 +170,86 @@ The pipe class exposes static member functions for writing a data word to a pipe
 
 Blocking and non-blocking forms of the read and write members are defined, with the form chosen based on overload resolution.
 
+Read/write methods take an ext::oneapi::properties argument, which can contain the latency control properties `latency_anchor_id` and/or `latency_constraint`.
+
+* `sycl::ext::intel::latency_anchor_id<N>`, where `N` is an integer: An ID to associate with the current read/write function call, which can then be referenced by other `latency_constraint` properties elsewhere in the program to define relative latency constaints. ID must be unique within the application, and a diagnostic is required if that condition is not met.
```

Comment:
> ID must be unique within the application, and a diagnostic is required if that condition is not met.

Sorry, I just noticed this.  Are you expecting the FPGA part of the compiler to issue this diagnostic?  A typical C++ compiler won't see all translation units together, so it won't be able to tell if the same ID is used in two different translation units.

Reply:
Yes, I expect the FPGA part to issue this diagnostic.